src/main/webapp/src/components/songInfo.js

```diff
@@ -144,7 +146,32 @@ class SongInfo extends React.Component {
       ],
     };
 
-    // TODO display entities with charts
+    function drawChart(element) {
```

Comment:
`drawChart` is a generic name. Please use a more meaningful one and/or add some docs to explain which chart we're drawing here (probably an entity pie chart) and giving details about how it's expected to look like